현정/5주차_2170.java

```diff
@@ -0,0 +1,76 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.*;
+
+public class BJ_2170 {
+    static int n;
+    static Point[] arr;
+
+    //객체 생성
+    static class Point{
+        int start; int end;
+        public Point(int x,int y) {
+            this.start = x;
+            this.end = y;
+        }
+    }
+
+    //연산부분
+    static int  calculate() {
+        //가장 첫번째 배열의 객체 받아오기
+        int s = arr[0].start;
+        int e = arr[0].end;
```

Comment:
```suggestion
        int s = arr[0].start;
        int e = arr[0].end;
```
start 하고 end인건 알겠는데 이것도 설명이 좀 부족하다고 생각해유
개인적인 생각으로는 어떻게 표현하던 변수명 한글자로는 그 의미를 다 표현하기가 어렵다고 생각하고

[변수명짓는 팁](https://rito15.github.io/posts/unity-naming-variables-neatly/)
쓸만한 링크 드릴테니까 한번쯤 고민해보시는게?